docs/servers.rst

```diff
@@ -352,3 +352,5 @@ the below commands. You should not be prompted for a passphrase
     app
     $ ssh <username>@<Monitor IP address> hostname
     mon
+
+If you have successfully connected to the app via SSH, the terminal output will be name of the app to which you have connected ('app' or 'mon') as shown above.
```

Comment:
Let's replace `app` with `server` in its first two uses, as the word app is also the hostname of one of the two servers. So we'd have: `If you have successfully connected to the server via SSH, the terminal output will be name of the server to which you have connected ('app' or 'mon') as shown above.`